lms/services/canvas.py

```diff
@@ -1,10 +1,11 @@
+from lms.services.canvas_api.client import CanvasAPIClient
 from lms.services.exceptions import CanvasAPIPermissionError, FileNotFoundInCourse
 
 
 class CanvasService:
     """A high level Canvas service."""
 
-    api = None
+    api: CanvasAPIClient = None  # type:ignore
```

Comment:
AFAICT this is only here for the tests benefrict. Ignore this declaration so all the code below assumes the right type.